app/assets/stylesheets/admin/_colors.scss

```diff
@@ -26,17 +26,19 @@ $background-gray: rgb(243, 243, 243);
 $gray2: rgb(230, 230, 230);
 $table-border-color: rgb(221, 221, 221); // Bootstrap Table Border #ddd
 $gray4: lightgrey; // rgb(211, 211, 211)
-$gray3: gray; // rgb(128, 128, 128)
+$gray3: gray; // rgb(128, 128, 128) and #8080
 $gray5: #f3f3f3;
 $gray6: hsl(0, 0%, 60%);
 $gray7: hsl(0, 0%, 80%);
+$gray8: #545454;  // Meets accessibility standard on $table-gray
+$gray9: #595959; // Meets accessibility standards on white background
 $body: #333;
 $black: black;
 
 // Color values from project step model
-$onTime: hsl(120, 73%, 57%);
-$superEarly: hsl(120, 41%, 47%);
-$barelyLate: hsl(56, 100%, 66%);
-$superLate: hsl(0, 74%, 54%);
```

Comment:
hsl is a bit non-standard in terms of using web development tools. Having the rgb values available makes things easier for future styling.

Reply:
Is there any reason not to just convert to RGB? I would be fine with that.

Reply:
The reason I used HSL here is that the project step model does calculations that require colors in HSL format. The colors are defined in that model as well as here. I think it's useful to have them defined in the same way in both places.

Reply:
I'll keep the values in hsl but keep rgb in the comments.